### PR TITLE
JobApplications

### PR DESCRIPTION
**Pull request by ReneSG**
# What does this PR do?
- Adds JobApplicationPolicy.
- Adds candidates list.
- Adds button to apply to a JobPosting.

image

image

image

### CODE REVIEW

File: app/views/students/_former_students_table.html.erb [hunk 1]
```diff
@@ -1,30 +1,32 @@
...
-            <div class="col"><a class="btn-small disabled table-button" table-button-tag="VER">Ver</a></div>
-            <div class="col"><a class="btn-small disabled table-button" table-button-tag="CV">CV</a></div>
-            <div class="col"><a class="btn-small disabled table-button" table-button-tag="ELIMINAR"">Eliminar</a></div>
-            <div class="col"><%= link_to 'Crear', new_with_former_student_path, :class=>"btn-small"%></div>
-            <div class="col"><%= link_to 'Importar', former_students_upload_path, :class=>"btn-small"%></div>
-            <div class="col"><%= link_to 'Descargar', former_students_path(format: :csv), :class=>"btn-small"%></div>
-        </div>
+      <span class="table-title"><%= table_title %></span>
+      <div class="actions">
+        <div class="col"><a class="btn-small disabled table-button" table-button-tag="VER">Ver</a></div>
```
Comment (eduardotru) [inline app/views/students/_former_students_table.html.erb:7]: Employers should not be able to see the former_students personal information

Comment (ReneSG) [reply app/views/students/_former_students_table.html.erb:7]: Done.

File: app/controllers/job_postings_controller.rb [hunk 2]
```diff
@@ -21,12 +17,21 @@ def index
     ) || return
 
     @job_postings = @filterrific.find.not_expired.paginate(page: params[:page])
-    @custom_paginate_renderer = custom_paginate_renderer
+  end
+
+  def my_job_postings
```
Comment (eduardotru) [inline app/controllers/job_postings_controller.rb:22]: should you authorize the company here?

Comment (ReneSG) [reply app/controllers/job_postings_controller.rb:22]: Done.

**Pull request merged**